pages/index.js

```diff
@@ -1,10 +1,12 @@
 import Head from 'next/head'
 import Contributors from '../components/Contributors';
+import Cards from '../components/Card';
```

Comment:
Sorry, one more small request. Can you update the name of the `Cards` component file to `Cards.js`?